lib/roo/excelx.rb

```diff
@@ -112,7 +110,10 @@ def initialize(filename_or_stream, options = {})
 
       if cell_max
         cell_count = ::Roo::Utils.num_cells_in_range(sheet_for(options.delete(:sheet)).dimensions)
-        raise ExceedsMaxError.new("Excel file exceeds cell maximum: #{cell_count} > #{cell_max}") if cell_count > cell_max
+        if cell_count > cell_max
+          raise Roo::TooManyCellsError,
```

Comment:
[use `fail` instead of `raise`](https://github.com/bbatsov/ruby-style-guide#exceptions)

    fail Roo::TooManyCellsError, ....